photutils/psf/models.py

```diff
@@ -497,8 +497,12 @@ def evaluate(self, x, y, flux, x_0, y_0, use_oversampling=True):
 
 class EPSFModel(FittableImageModel):
     """
-    A subclass of `FittableImageModel`. A fittable ePSF model.
+    A class that models an effective PSF (ePSF). This is a subclass of
+    `FittableImageModel`, and is very similar but contains a few additional
+    parameters relevant specifically for ePSF's.
```

Comment:
```suggestion
    parameters relevant specifically for ePSFs.
```

My only comment is a very minor grammar fix.